src/diffusers/pipelines/controlnet_sd3/pipeline_stable_diffusion_3_controlnet.py

```diff
@@ -858,6 +858,12 @@ def __call__(
         height = height or self.default_sample_size * self.vae_scale_factor
         width = width or self.default_sample_size * self.vae_scale_factor
 
+        controlnet_config = (
+            self.controlnet.config
+            if isinstance(self.controlnet, SD3ControlNetModel)
+            else self.controlnet.nets[0].config
```

Comment:
I guess this is okay for now but could there be a case where we may have incompatibility configs when `len(self.controlnet.nets) > 1`? I guess we will know when we will know.